extensions/parquet/compression/src/main/java/io/deephaven/parquet/compress/DeephavenCompressorAdapterFactory.java

```diff
@@ -151,17 +148,10 @@ public BytesInput decompress(final InputStream inputStream, final int compressed
             } else {
                 decompressor = null;
             }
-            try {
-                // Note that we don't close the decompressed stream because doing so may return the decompressor to the
-                // pool
-                final InputStream buffered = ByteStreams.limit(IOUtils.buffer(inputStream), compressedSize);
-                final CompressionInputStream decompressed = compressionCodec.createInputStream(buffered, decompressor);
-                return BytesInput.copy(BytesInput.from(decompressed, uncompressedSize));
-            } finally {
-                if (decompressor != null) {
-                    decompressor.reset();
-                }
-            }
+            // Note that we don't want the caller to close the decompressed stream because doing so may return the
+            // decompressor to the pool.
+            final InputStream buffered = ByteStreams.limit(inputStream, compressedSize);
```

Comment:
`buffered` may be a bad name for this input stream given the buffer was removed? Are we assuming the inputStream is already buffered, or assuming that `createInputStream` will buffer if necessary?